conan/tools/system/package_manager.py

```diff
@@ -56,9 +55,11 @@ def get_default_tool(self):
                 if d in os_name:
                     return tool
 
-    def get_package_name(self, package):
-        # TODO: should we only add the arch if cross-building?
-        if self._arch in self._arch_names and cross_building(self._conanfile):
+    def get_package_name(self, package, host_tool=True):        
+        # Only if the package is for building, for example a library,
+        # we should add the destination arch (important especially in cross building).
+        # If the package is a tool that should be installed on the host we should not add the arch.
+        if self._arch in self._arch_names and not host_tool:
```

Comment:
I though that if the package should be linked to the build package maybe it a good practice to add the arch anyway. But is seems that it will not make any dramatic different so I can keep this condition :)

Reply:
We usually tend to favor commands as close to the developer experience as possible. So if a user uses ``apt install xxxx`` without architecture as default (and they do), it is slightly better that the conan command does the same.

Reply:
Sounds logical :+1: I keep the condition